saul-core/src/main/java/edu/illinois/cs/cogcomp/saul/learn/SaulWekaWrapper.java

```diff
@@ -0,0 +1,587 @@
+package edu.illinois.cs.cogcomp.saul.learn;
+
+import akka.japi.tuple.Tuple4;
```

Comment:
I think this namespace is accessible in saul-core because of a dependency on Play Framework. Can you use something else here? Maybe a private class with 4 members. 

@danyaljj Thoughts on this namespace?